### ISSUE TITLE
Certain road pieces stop auto walk. 

### ISSUE THREAD

**Issue by Tarr-OHOL** [jasonrohrer/OneLifeData7#856]
https://onetech.info/4453-North-East-West-Road-Tee-A-road4426
https://onetech.info/4452-North-South-West-Road-Tee-D-road4426
https://onetech.info/4450-North-West-Road-Elbow-road4426-noCover
https://onetech.info/4449-North-East-Road-Elbow-road4426-noCover
https://onetech.info/4451-South-East-West-Road-Tee-C-road4426
https://onetech.info/4431-South-East-Road-Elbow-road4426-noCover
https://onetech.info/4430-Paved-Crossroad-road4426
https://onetech.info/4428-North-South-East-Road-Tee-B-road4426

Basically any time you try to use the fast walk on any tiles besides the horizontal and vertical you will stop walking or proceed to walk onto another set of these roads if they're directly next to each other. 
https://onetech.info/4426-Paved-Road-East-West
https://onetech.info/4427-Paved-Road-North-South-road4426

**Comment by TwistedHBG** [comment]
Works fine for me, I haven't had any problems auto-running on corners. Some people in Discord mentioned it's a bug related to the zoom mod they're using.

**Comment by Tarr-OHOL** [comment]
Closing as this seems to work on both modded and normal clients now.
